salt/modules/system.py

```diff
@@ -428,3 +428,87 @@ def tzname(self, dt):  # pylint: disable=W0613
 
     def dst(self, dt):  # pylint: disable=W0613
         return timedelta(0)
+
+
+def _strip_quotes(str_q):
+    """
```

Comment:
Could you please change these docstring quotes to single-quotes, per our style guide? https://docs.saltstack.com/en/latest/topics/development/conventions/style.html